activerecord/test/cases/serialized_attribute_test.rb

```diff
@@ -367,7 +371,7 @@ def deserialize(value)
   end
 
   def test_serialized_attribute_works_under_concurrent_initial_access
-    model = Topic.dup
+    model = ::Topic.dup
```

Comment:
Why is this using the original class?

Reply:
It dupes it, so the original class is unchanged. Since duping seems to have the same affect and doesn't need type columns, I'll change my code to just use a dup of `Topic` everywhere.